content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -719,9 +719,13 @@ is turned on.
   more than one default is specified, the admission plugin forbids the creation of
   all PVCs.
 * If the admission plugin is turned off, there is no notion of a default
-  StorageClass. All PVCs that have no `storageClassName` can be bound only to PVs that
-  have no class. In this case, the PVCs that have no `storageClassName` are treated the
-  same way as PVCs that have their `storageClassName` set to `""`.
+  StorageClass. All PVCs that have `storageClassName` set to `""` can be
+  bound only to PVs that have `storageClassName` also set to `""`.
+  However, PVCs with missing `storageClassName` can be updated later once
+  default StorageClass becomes available. If the PVC gets updated it will no
+  longer bind to PVs that have `storageClassName` also set to `""`.
+
+See [retroactive StorageClass assignment](#retroactive-storageclass-assignment) for more details.
```

Comment:
```suggestion
See [retroactive default StorageClass assignment](#retroactive-default-storageclass-assignment) for more details.
```
The anchor needs to be modified too according to the changed head title :)

Also, how about changing all `retroactive StorageClass assignment` to `retroactive default StorageClass assignment` which is the exact name of KEP ? :)

Reply:
Sure, no hard opinion on the naming :)